### PR TITLE
Add CIS iptables rules

### PR DESCRIPTION
**Pull request by dodys**
#### Description:

- Add new rules:
  - package_ufw_removed
  - package_iptables-persistent_installed
  - packages_iptables-persistent_removed

- Add existing rules to Ubuntu CIS profile:
  - set_iptables_default_rule
  - set_ip6tables_default_rule
  - set_loopback_traffic
  - set_ipv6_loopback_traffic
 
- Remove Ubuntu from service_iptables_enabled

#### Rationale:

- Those rules are needed for CIS on Ubuntu 22.04 and/or 20.04.

### CODE REVIEW

File: linux_os/guide/system/network/network-iptables/package_iptables-persistent_removed/rule.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+severity: medium
+
+references:
+    cis@ubuntu2004: 3.5.1.2
+    cis@ubuntu2204: 3.5.1.2
+
+ocil_clause: 'the package is installed'
+
+ocil: '{{{ ocil_package(package="iptables-persisten") }}}'
```
Comment (Mab879) [inline linux_os/guide/system/network/network-iptables/package_iptables-persistent_removed/rule.yml:22]:
```suggestion
ocil: '{{{ ocil_package(package="iptables-persistent") }}}'
```

File: linux_os/guide/system/network/network-iptables/iptables_activation/set_loopback_traffic/rule.yml [hunk 2]
```diff
@@ -23,21 +23,24 @@ identifiers:
...
 ocil_clause: 'loopback traffic is not configured'
 
 ocil: |-
-    Verify that the loopback interface is configured:
+    Run the following commands and verify output:
+    <pre>
+    # iptables -L INPUT -v -n | grep lo | grep ACCEPT
+    </pre>
     <pre>
-    # nft list ruleset | awk '/hook input/,/}/' | grep 'ip saddr'
```
Comment (Mab879) [inline linux_os/guide/system/network/network-iptables/iptables_activation/set_loopback_traffic/rule.yml:38]: Switching to iptables here concerns me. I believe we should add Jinja condition for these products that want iptables.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Thanks for the PR. There are a couple of items I found during my review.

File: linux_os/guide/system/network/network-iptables/iptables_activation/set_loopback_traffic/rule.yml (hunk 2 above)
Comment (dodys) [reply linux_os/guide/system/network/network-iptables/iptables_activation/set_loopback_traffic/rule.yml:38]: shouldn't nftables rules be under: `linux_os/guide/system/network/network-nftables/` ?

The bash itself is about iptables and not nft

Comment (Mab879) [reply linux_os/guide/system/network/network-iptables/iptables_activation/set_loopback_traffic/rule.yml:38]: I agree; we will keep `iptables` here.

**Review by Mab879 — APPROVED** [review]
Thanks for the PR! Override CODEOWNERS since @dodys can't merge this PR.

**Comment by Mab879** [comment]
Automatus failures are expected due to the prodtype.

**Pull request merged**